e2e/scripts/run-e2e.sh

```diff
@@ -30,7 +30,7 @@ while [ $secs -ge 0 ]; do
 done
 
 printf "\nRunning tests...\n"
-SYNC=true go test ./e2e/
+SYNC=true go test -p 1 ./e2e/
```

Comment:
I'm not sure this change is required, but I saw this flag being set in other places we were calling `go test`, so figured it should probably be set here too.